Cargo.toml

```diff
@@ -16,19 +16,18 @@ exclude = [ "tests/*", "derive/*" ]
 panic = "abort"
 
 [dependencies]
-tiny-keccak = { version = "1.3", default-features = false }
-rustc-hex = { version = "1.0", optional = true }
+rustc-hex = { version = "1.0.0", optional = true }
 
 [dependencies.bigint]
-version = "4"
+version = "4.4.0"
```

Comment:
why do we need restrict to 4.4?

Reply:
It's a carret requirement. It doesn't restrict to 4.4, it only says that version should be higher.

It's specified by `cargo upgrade` by default, which is a good tool I suggest to try. Having more precise versions may help having more stable builds.